app/src/main/java/org/sil/storyproducer/tools/media/story/AutoStoryMaker.kt

```diff
@@ -141,9 +144,22 @@ class AutoStoryMaker(private val context: Context) : Thread(), Closeable {
 
         var lastSoundtrack = ""
         var lastSoundtrackVolume = 0.0f
+        val slides = Workspace.activeStory.slides.toCollection(mutableListOf())
         var iSlide = 0
-        while (iSlide < (Workspace.activeStory.slides.size)) {
-            val slide = Workspace.activeStory.slides[iSlide++]
+
+        // Create Local Credits Slide
+        var slide = Slide()
+        slide.slideType = SlideType.COPYRIGHT
+        slide.content = Workspace.activeStory.localCredits + "\n" +
+                    "This video is licensed under a Creative Commons Attribution" +
```

Comment:
Hardcoded strings will need to be moved to where they can be properly localized.